packages/core/package.json

```
@@ -21,7 +21,7 @@
   "peerDependencies": {
     "@angular/compiler": "0.0.0-PLACEHOLDER",
     "rxjs": "^6.5.3 || ^7.4.0",
-    "zone.js": "~0.15.0"
+    "zone.js": "~0.16.0"
```

We need to support 0.15.0 otherwise it's a breaking change. 

```suggestion
    "zone.js": "~0.15.0 || ~0.16.0"
```